CHANGELOG.md

```diff
@@ -112,6 +114,10 @@ All notable changes to "Azure Toolkit for IntelliJ IDEA" will be documented in t
   - [3.0.7](#307)
   - [3.0.6](#306)
 
+## 3.92.0
+- Added Managed identity support for Azure Functions.
```

Comment:
https://github.com/microsoft/azure-tools-for-java/pull/8497 fix support for swap